os/hal/ports/SN32/SN32F260/sn32_registry.h

```diff
@@ -25,12 +25,7 @@
 #ifndef SN32_REGISTRY_H
 #define SN32_REGISTRY_H
 
-// There is some usage of SN32F24xx in:
-// - qmk_firmware/tmk_core/protocol/chibios/main.c
-// - qmk_firmware/tmk_core/common/chibios/suspend.c
-// - qmk_firmware/tmk_core/common/chibios/sleep_led.c
-// But it is me not really clear what this does.
-// SN32F24x and SN32F24xB both have this defines, so also define it here I guess
+/* Common identifier of all SN32F24xx devices.*/
```

Comment:
But this is a file for a 260 device, wouldn't `SN32F2xx` be a better name?

Reply:
this is mostly used atm as a pointer to the hal. A `SN32F2xx` name would imply support for all of these
![image](https://user-images.githubusercontent.com/1979042/142277189-16bc5772-ca35-46b2-bf74-ee6bb4012f82.png)

So far it was an arbitrary selection. We should probably discuss on standardizing this.